CMakeLists.txt

```diff
@@ -42,6 +42,9 @@ gz_find_package(gz-common5
   REQUIRED_BY heightmap mesh dartsim tpe tpelib bullet)
 set(GZ_COMMON_VER ${gz-common5_VERSION_MAJOR})
 
+# This is only used for test support
+gz_find_package(gz-common5 COMPONENTS testing)
```

Comment:
I think we should either find with `REQUIRED` or put logic in `test/CMakeLists.txt` to fail with a coherent error message if it is not found

Reply:
I'm okay with `REQUIRED`. 

I think the common way to handle this would be to put it inside of a block protected by `BUILD_TESTING`, but I don't think that gz-cmake is currently equipped to do that.